hawc/apps/riskofbias/serializers.py

```diff
@@ -167,10 +170,11 @@ def to_representation(self, instance):
 class RiskOfBiasScoreOverrideCreateSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.RiskOfBiasScore
-        fields = ("id", "riskofbias", "metric")
+        fields = ("id", "score", "riskofbias", "metric")
 
     def create(self, validated_data):
         override = models.RiskOfBiasScore.objects.create(
+            score=validated_data["score"],
```

Comment:
do we still need to pass this on create? I would think it'd just create the default on the backend using `RESPONSES_VALUES_DEFAULT`

Reply:
Right now our default score is 0, not whatever the metric default is. On save our score [throws an error if it is not a valid score for its metric](https://github.com/shapiromatron/hawc/blob/cd12d60d973ec0a78356c510819559e0fae0e3aa/hawc/apps/riskofbias/models.py#L381). Do you think it would be better to just set the metric default here instead? That way we wouldn't have to pass a default in on create.

Reply:
yep, lets set the default for the metric here! If i recall; this is triggered when the user presses the "Create new override" button, is that right? If so, then I think it'd be perfectly logical to create it here.

I'm thinking you're right too... given this should we remove the default score of 0?

Reply:
That's correct re: "Create new override". The only harm in keeping the default score 0 is for cases where we bypass the model save function (`bulk_create` etc) and we're having the scores set to 0 instead of having errors thrown. It would probably be more correct to remove the default, but we'd have another migration as a tradeoff. What do you think?

Reply:
Seems worth it, lets' do it!